Pipfile

```diff
@@ -6,7 +6,7 @@ name = "pypi"
 [packages]
 CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "master"}
 RapidProTools = {editable = true,git = "https://www.github.com/AfricasVoices/RapidProTools",ref = "master"}
-CodaV2PythonClient = {editable = true,git = "https://www.github.com/AfricasVoices/CodaV2PythonClient",ref = "msg-get-add-experiment"}
+CodaV2PythonClient = {editable = true,git = "https://www.github.com/AfricasVoices/CodaV2PythonClient",ref = "master"}
```

Comment:
Please setup a version bump for CodaV2PythonClient e.g "0.0.1". Then update here and also run pipenv lock for it.

Reply:
Updated, thanks.